README.md

```diff
@@ -63,5 +63,30 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 
 Run the following
 ```bash
-$ poetry run pytest
+poetry run pytest
+```
+
+## Running in Docker
+
+Build the Docker images by doing:
+```bash
+docker build --target development --tag todo-app:dev .
+docker build --target production --tag todo-app:prod .
+```
+
+You can run the production image by doing
+```bash
+docker run --env-file .env -p 8000:80 todo-app:prod
```

Comment:
Very much optional but I like to add an additional `-it` flag so that I can more easily kill containers from the terminal with ctrl+c